Rubberduck.Core/UI/Controls/SearchBox.xaml.cs

```diff
@@ -65,7 +54,7 @@ public string Hint
             }
         }
 
-        public ICommand ClearSearchCommand { get => new DelegateCommand(LogManager.GetCurrentClassLogger(), (arg) => Text = ""); }
+        public ICommand ClearSearchCommand => new DelegateCommand(LogManager.GetCurrentClassLogger(), (arg) => Text = "");
```

Comment:
:+1:

Reply:
D'oh. Just saw this again. I ---could--- should replace `""` with `string.Empty`.